tests/src/Kernel/Entity/SelectionHandlerTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\SelectionHandlerTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Component\Utility\Unicode;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\node\Entity\NodeType;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests entity reference selection plugins.
+ *
+ * @group og
+ */
+class SelectionHandlerTest extends KernelTestBase {
```

Comment:
This class doesn't test much yet, I guess we are expanding this in a follow up PR?